### PR TITLE
style: Minor nits on the alignment properties.

### PR DESCRIPTION
**Pull request by emilio**
I'm going to touch this in a bit, let's do it a bit less painful.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19850)

### CODE REVIEW

File: components/style/gecko/generated/structs.rs [hunk 1]
```diff
@@ -26590,8 +26590,8 @@ pub mod root {
         pub static mut nsCSSProps_kAutoCompletionAlignItems: [root::nsCSSProps_KTableEntry; 0usize];
     }
     extern "C" {
-        #[link_name = "\u{1}_ZN10nsCSSProps34kAutoCompletionAlignJustifyContentE"]
-        pub static mut nsCSSProps_kAutoCompletionAlignJustifyContent:
+        #[link_name = "\u{1}_ZN10nsCSSProps34kAutoCompletionContentDistributionE"]
+        pub static mut nsCSSProps_kAutoCompletionContentDistribution:
```
Comment (nox) [inline components/style/gecko/generated/structs.rs]: Is this change intended? Please ELI5.

**Review by nox — APPROVED** [review]
r=me with an explanation in the ticket, aren't the generated files you changed generated on the Gecko side?

File: components/style/gecko/generated/structs.rs (hunk 1 above)
Comment (emilio) [reply components/style/gecko/generated/structs.rs]: Yeah, it is not, but it's harmless anyway.

**Comment by emilio** [comment]
@bors-servo r=nox

 * I reverted the generated changes.

**Comment by emilio** [comment]
@bors-servo retry

 * ARM build failed to checkout?

**Comment by emilio** [comment]
@bors-servo retry 

> command timed out: 1200 seconds without output running ['./mach', 'build', '--rel', '--target=arm-unknown-linux-gnueabihf'], attempting to kill
